mapzap/models/staging/stg_dcp__zap_bbls.sql

```diff
@@ -0,0 +1,21 @@
+with source_zap_project_bbls as (
+
+    select * from {{ source('zap_bbls', '20230526') }}
```

Comment:
What's producing this table? Other code in this repo?

Reply:
Sorry you're getting a lot of just general knowledge questions and not just code review! But just trying to understand what specifically is happening in all of this

Reply:
haha worries! this table is a source that's expected to exist already for the build to use (the result of EL before the T)

all source tables are declared in [`_dcp_source.yml`](https://github.com/NYCPlanning/db-zap-opendata/blob/build-map-zap/mapzap/models/staging/_dcp_source.yml) and (by convention) are references using `{{ source(name, table) }}` rather than `{{ ref ... }}`

as I'm typing this, I think the advantages to using `{{ source ... }}` are that the metadata then knows it's a source and not a model and if it isn't configured in `_dcp_source.yml`, it can't be found and has an informative error

Reply:
I actually mean specifically how is this source generated? Are we producing it? The bigquery table

Reply:
ah, yea we produce it via this repo's weekly export of ZAP CRM data that I improved in https://github.com/NYCPlanning/db-zap-opendata/pull/42 last week